### DISCUSSION TITLE
How do I login with vercel cli?

### DISCUSSION THREAD

**Discussion by harm-meijer** [vercel/vercel#5468] [Help] [answered]
I do `vercel login` and it asks me for my email, when I login through the website I use github and don't know what email to provide here.

**Comment by harm-meijer** [comment, marked as answer]
I got the email from the profile, recently our company changed the email top level domain so could not login with my current email.

**Reply by manga4th** [reply]
is this the windows command or macOS

**Reply by amyegan** [reply]
Hi @manga4th! This is a fairly old discussion. If you are currently having trouble with [Vercel CLI](https://vercel.com/docs/cli) and [`vercel login`](https://vercel.com/docs/cli#commands/login), please [create a new discussion](https://github.com/vercel/vercel/discussions/new) and we can get it working for you!
